packages/health/example/android/app/src/main/kotlin/cachet/plugins/example_app/MainActivity.kt

```diff
@@ -2,7 +2,7 @@ package cachet.plugins.example_app
 
 import android.os.Bundle
 
-import io.flutter.embedding.android.FlutterActivity
+import io.flutter.embedding.android.FlutterFragmentActivity
```

Comment:
Add documentation for this step as well since the developer has to modify their Android application for this plugin to work now.